packages/frontend-2/components/dashboard/Blog/Wrapper.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <section v-if="!error">
+    <h2 class="text-heading-sm text-foreground-2 mb-4">Blog</h2>
+    <div class="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4 gap-4">
+      <DashboardBlogCard
+        v-for="webflowItem in webflowItems"
+        :key="webflowItem.id"
+        :webflow-item="webflowItem"
+      />
+    </div>
+  </section>
+  <section v-else />
+</template>
+
+<script setup lang="ts">
+import type { WebflowItem } from '~/lib/dashboard/helpers/types'
+
+const logger = useLogger()
+
+const { data: webflowData, error } = await useFetch<{
```

Comment:
Shouldn't you use useAsyncData instead, for proper caching and stuff? useFetch works, but may be too low level, it probably refetches the API every time this component initializes